docs/reference/api.md

```diff
@@ -0,0 +1,214 @@
+# API
+
+Pebble exposes an HTTP API that remote clients can use to interact with the daemon. The API has endpoints for starting and stopping services, adding configuration layers to the plan, and so on.
+
+The API uses HTTP over a Unix socket, with access to the API controlled by user ID. If `pebble run` is started with the `--http <address>` option, Pebble exposes a limited set of open-access endpoints (see {ref}`api-access-levels`) using the given TCP address.
+
+## Structure of the API
+
+Here is a breakdown of the general structure of most API endpoints in Pebble.
+
+### API endpoint URL
+
+Pebble API uses standard HTTP URLs to get actions and resources. The URLs are made up of a base URL followed by a path and optional query parameters.
+
+- Base URL: The consistent foundation of all API endpoints (for example, `http://localhost:4000/v1/`).
+- Path: Specifies the resource or collection being accessed (for example, `/changes`, `/services`). Paths are hierarchical and use forward slashes as separators.
```

Comment:
I think we should always include `/v1` in paths, since our concept of base URL is a bit different from normal